src/components/HeaderFeedback.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import md5 from 'crypto-js/md5';
+
+class Header extends Component {
+  getScore() {
+    const currentScore = JSON.parse(localStorage.getItem('score'));
+    return currentScore;
+  }
```

Comment:
```suggestion
class Header extends Component {
  getScore() {
    const { score } = JSON.parse(localStorage.getItem('state')).player;
    return { score };
  }
```
Para acessar a chave 'score', deve ser feito dessa forma